packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -183,12 +205,27 @@ it('Create a directly funded channel between two wallets ', async () => {
     turnNum: 4,
   });
 
-  expect(await closeCompletedA).toMatchObject({channelId, objectiveType: 'CloseChannel'});
-  expect(await closeCompletedB).toMatchObject({channelId, objectiveType: 'CloseChannel'});
+  expect(await closeCompletedA).toMatchObject({
+    channelId,
+    objectiveType: 'CloseChannel',
+  });
+  expect(await closeCompletedB).toMatchObject({
+    channelId,
+    objectiveType: 'CloseChannel',
+  });
 
   const aBalanceFinal = await getBalance(aAddress);
   const bBalanceFinal = await getBalance(bAddress);
 
   expect(BN.sub(aBalanceFinal, aBalanceInit)).toEqual(aFunding);
   expect(BN.sub(bBalanceFinal, bBalanceInit)).toEqual(bFunding);
-}, 20_000);
+
+  // TODO: remove this
```

Comment:
Let's file an issue for this?

Reply:
Should be covered by https://github.com/statechannels/statechannels/issues/2928. Updated the description of that issue.